### DISCUSSION TITLE
using  rounded-l and rounded-r to same element is not working

### DISCUSSION THREAD

**Discussion by 18121A05L2** [tailwindlabs/tailwindcss#9350] [Help]
**What version of Tailwind CSS are you using?**

For example: v3.1.2

**What build tool (or framework if it abstracts the build tool) are you using?**

 postcss-cli 8.4.14, Next.js 10.0.9 (latest ), webpack 5.28.0

**What version of Node.js are you using?**

For example: v16.0.0

**What browser are you using?**

Chrome, Safari

**What operating system are you using?**

Windows

**Reproduction URL**

A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways. Unfortunately we can't provide support without a reproduction, and your issue will be closed with no comment if this is not provided.

**Describe your issue**

Hello team using both rounded-r-[10px] and rounded-l-full , is not working at the same time , can u please look into this
![Screenshot (4388)](https://user-images.githubusercontent.com/59156926/190842915-da01fe6f-47c6-41d2-aba7-a4a6c266a3f7.png)

**Comment by exSnake** [comment]
The problem seems to be the rounded-l-full, if you set one side to full the other side will be borderless
